flink-connector/flink/src/main/java/org/apache/gravitino/flink/connector/catalog/BaseCatalog.java

```diff
@@ -521,6 +552,11 @@ protected CatalogBaseTable toFlinkTable(Table table) {
           .column(column.name(), column.nullable() ? flinkType.nullable() : flinkType.notNull())
           .withComment(column.comment());
     }
+    Index[] indices = table.index();
+    if (indices != null && indices.length == 1) {
```

Comment:
So .... this means the primary key always has a SINGLE column, right?

Reply:
No. His meaning is that only one primary key is supported. The Index object also stores field information and there can be multiple fields.
![image](https://github.com/user-attachments/assets/22e2d49a-edae-40f2-aad1-6bb9a6b2f1f4)